build.rs

```diff
@@ -70,6 +70,17 @@ pub fn main() {
                 )
                 .unwrap();
             }
+            "uu-tail" | "tail" => {
+                phf_map.entry("tail", "(tail::uumain, tail::args::uu_app)");
+                tf.write_all(
+                    format!(
+                        "#[path=\"{dir}/test_tail.rs\"]\nmod test_tail;\n",
+                        dir = util_tests_dir,
+                    )
+                    .as_bytes(),
+                )
+                .unwrap();
+            }
```

Comment:
Maybe it makes more sense to re-export `uu_app` from `tail`? I.e.

```rust
pub use args::uu_app;
```

I do like that you split the args into a separate file though!